src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,178 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static final String FIFTEEN_CHARS_LOGIN = "Traineeeeeeeeee";
+    private static final String SIX_CHARS_LOGIN = "Taylor";
```

Comment:
Please add some context to names, six chars and fifteen chars and two chars is equal if developer who reads test don`t know about them a lot, so names live VALID_LOGIN/INVALID_LOGIN preferable